dashboard-for-web/articles/web-dashboard-designer-mode/data-analysis/aggregations.md

```diff
@@ -30,6 +30,6 @@ These functions can be used for all types of numeric fields. After creating such
 
```

Comment:
Line 17: Returns a number of non-null objects in a collection.

Can we update this to the following?

`Returns the number of non-null objects in a collection.`